File: 0014-longest-common-prefix/linear-comparison/linear.cs
public class Solution
{
    public string LongestCommonPrefix(string[] strs)
    {
        if (strs.Length == 0)
        {
            return "";
        }
        else
        {
            for (int j = 0; j < strs[0].Length; j++)
            {
                for (int i = 1; i < strs.Length; i++)
                {
                    if ((j == strs[i].Length) || (strs[i][j] != strs[0][j]))  //Prefix no longer matches
                    {
                        // Return substring
                        return strs[0].Substring(0, j);
                    }
                }
            }

            // Default return
            return strs[0];
        }
    }
}

File: 0014-longest-common-prefix/linear-comparison/linear.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        if not strs:
            return ""
        else:
            for j in range(len(strs[0])):
                for i in range(1, len(strs)):
                    if ((j == len(strs[i])) or (strs[i][j] != strs[0][j])): # Prefix no longer matches
                        # Return substring
                        return strs[0][:j]
            # Default return
            return strs[0]